### PR TITLE
Refactor(api): Add offset to maintenance position

### PR DESCRIPTION
**Pull request by TamarZanzouri**
# Overview

closes https://opentrons.atlassian.net/browse/RLIQ-326, https://opentrons.atlassian.net/browse/RLIQ-325, https://opentrons.atlassian.net/browse/RLIQ-324.
Change slot for maintenance position to slot 1 and add offsets for the z and y. 

# Test Plan

# Changelog

- changed slot for attach a pipette to slot_1.
- added offsets from center slot supplied by @Laura-Danielle . still need to test on a robot to make sure its correct. 

# Review requests

Does these changes make sense?

# Risk assessment

low. this effects the attach location. should be tested to make sure the numbers are correct.

### CODE REVIEW

File: api/src/opentrons/protocol_engine/state/labware.py [hunk 1]
```diff
@@ -59,7 +59,9 @@
     "opentrons/usascientific_96_wellplate_2.4ml_deep/1",
 }
 
-_INSTRUMENT_ATTACH_SLOT = DeckSlotName.SLOT_2
+_INSTRUMENT_ATTACH_SLOT = DeckSlotName.SLOT_1
+# These offsets are based on testing attach flows with 8/1 channel pipettes
+_INSTRUMENT_ATTACH_OFFSET = Point(y=10, z=400)
```
Comment (TamarZanzouri) [inline api/src/opentrons/protocol_engine/state/labware.py]: do we want to move this offset to the move_to_maintanence_position command?

**Comment by TamarZanzouri** [comment]
tested on Felx. Position works perfect

File: api/src/opentrons/protocol_engine/state/labware.py (hunk 1 above)
Comment (Laura-Danielle) [reply api/src/opentrons/protocol_engine/state/labware.py]: Yes I think so.

**Review by Laura-Danielle — APPROVED** [review]
Thank you for this :)

File: api/src/opentrons/protocol_engine/state/labware.py [hunk 2]
```diff
@@ -596,16 +598,16 @@ def raise_if_labware_in_location(
                     f"Labware {labware.loadName} is already present at {location}."
                 )
 
-    def get_calibration_coordinates(self, current_z_position: float) -> Point:
+    def get_calibration_coordinates(self) -> Point:
         """Get calibration critical point and target position."""
         target_center = self.get_slot_center_position(_INSTRUMENT_ATTACH_SLOT)
         # TODO (tz, 11-30-22): These coordinates wont work for OT-2. We will need to apply offsets after
```
Comment (TamarZanzouri) [inline api/src/opentrons/protocol_engine/state/labware.py:602]: @Laura-Danielle is this comment still relevant? we just spoke about using this just for the OT3

Comment (Laura-Danielle) [reply api/src/opentrons/protocol_engine/state/labware.py:602]: I think we have the check in place to make sure it's an ot3 hardware, right?

Comment (TamarZanzouri) [reply api/src/opentrons/protocol_engine/state/labware.py:602]: so why not guard that its an ot3 command only?

**Review by SyntaxColoring — APPROVED** [review]
Code changes make sense to me, assuming this has been tested on a robot to your satisfaction! 📏

**Pull request merged**